packages/template-retail-react-app/config/default.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
                 callbackURI: 'https://webhook.site/27761b71-50c1-4097-a600-21a3b89a546c'
             },
             social: {
-                enabled: false,
+                enabled: true,
```

Comment:
should this be default to false?

Reply:
Good callout. For those customers using extensibility you wouldn't want any kind of visual surprises to their website after simply updating their template. it's possible that you would't if you don't have IDPs defined, but best practice is to not flip this config.